web/themes/custom/cloudy/src/components/morsel/morsel.twig

```diff
@@ -7,40 +22,20 @@
         classes: ['mb-2'],
       } only %}
     {% endif %}
-    {% if overline %}
-      {% include "@components/overline/overline.twig" with {
-        text: overline,
+    <div class="morsel__heading">
+      {% include "@components/heading/heading.twig" with {
+        url: url,
+        level: 3,
```

Comment:
Teaser and Search need to be an H2 and Featured needs to be an H3. While these are three different implementations of morsel, the heading level is not the same for every level of morsel. Heading level should not be used exclusively to set size as the headings on the page are how screen readers and keyboard navigators jump from section to section and title to title.

Also, I'm pretty sure we wanted a pretty fixed heading "size" for morsel that was a little smaller than the h3. Ideally, we would pass through the sizing class separate from the heading level at the level of the morsel component.

Reply:
I'll go ahead and add a `heading_level` prop to morsel so that we can differentiate at the pattern level. Good call! :-)